project-base/src/Shopsys/ShopBundle/Resources/views/Grunt/gruntfile.js.twig

```diff
@@ -30,7 +30,7 @@ module.exports = function(grunt) {
 
                     modifyVars: {
                         frameworkResourcesDirectory: '{{ frameworkResourcesDirectory|raw }}',
-                    }
+                }
```

Comment:
Why this one was moved? It is ending bracket for `modifyVars`.

Reply:
It was propably done during rebase and i missed it. Fixed.